.gitignore

```diff
@@ -19,3 +19,5 @@ src/static/riot.tag
 src/static/generated/*
 
 db.sqlite3
+competitions_v2
+postgres
```

Comment:
What are these?

Reply:
Pulled from `.gitignore` and added to my personal `.gitignore_global`